src/messages/MOSDScrubReserve.h

```diff
@@ -32,6 +32,9 @@ class MOSDScrubReserve : public MOSDFastDispatchOp {
   };
   int32_t type;
   pg_shard_t from;
+  /// Urgency of the scrub session itself. 0 for periodic scrubs.
+  /// Higher for user-initiated and other 'must' scrubs.
+  uint32_t urgency = 0;
```

Comment:
Define an enum for this and use for this and all function/method params as well as class members/variables.

Reply:
There is such an enum, of course:

```
enum class urgency_t {
  off,
  penalized,
  periodic_regular,
  overdue,
  operator_requested,
  must,
  after_repair,
};

```

But it is part of the scheduling PR, which is delayed.
I cannot introduce it now.

Reply:
(BTW - just noticed that this isn't the latest version of that enum. All the more reason why not use that draft -  now).